src/Findologic/MainVariant.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Findologic;
+
+final class MainVariant
```

Comment:
Feels like these constants should belong in the `\FINDOLOGIC\FinSearch\Struct\Config` class.

Reply:
We have done a similar thing for the filter position, so was just following the same pattern.

Reply:
Fine :+1: